### ISSUE TITLE
Training the model on NYUv2

### ISSUE THREAD

**Issue by isunLt** [YkiWu/EmbodiedOcc#8]
Thanks for the great work. I'm trying to train the model on NYUv2. However, I only get `15.56` for mIoU. What might be the reason? How can I improve the performance?
```bash
2025/06/27 16:35:24 - mmengine - INFO - Current val iou of sem_cls is [ 0.889  0.061  0.472  0.025  0.083  0.085  0.327  0.247  0.060  0.128
  0.149  0.075]
2025/06/27 16:35:24 - mmengine - INFO - Current val iou of sem is 0.15560498508398263
2025/06/27 16:35:24 - mmengine - INFO - Current val iou of geo is 0.25096069999079246
```

**Comment by YkiWu** [comment]
> Thanks for the great work. I'm trying to train the model on NYUv2. However, I only get `15.56` for mIoU. What might be the reason? How can I improve the performance?
> 
> 2025/06/27 16:35:24 - mmengine - INFO - Current val iou of sem_cls is [ 0.889  0.061  0.472  0.025  0.083  0.085  0.327  0.247  0.060  0.128
>   0.149  0.075]
> 2025/06/27 16:35:24 - mmengine - INFO - Current val iou of sem is 0.15560498508398263
> 2025/06/27 16:35:24 - mmengine - INFO - Current val iou of geo is 0.25096069999079246

We did not conduct experiments on NYUv2 since it only contains individual images and is not compatible with our embodied setting.
For training local occupancy prediction on NYUv2, I suggest considering the following directions:
1. Depth-aware branch: The depth prediction module we provide is tailored for the ScanNet dataset, so it may require adaptation for use with NYUv2.
2. Dataset-specific characteristics: Different datasets have different intrinsic properties, such as category distribution and so on. You can try adjusting the Gaussian parameters, the loss weightings, and other related settings to better suit this dataset.

**Comment by isunLt** [comment]
> > Thanks for the great work. I'm trying to train the model on NYUv2. However, I only get `15.56` for mIoU. What might be the reason? How can I improve the performance?
> > 2025/06/27 16:35:24 - mmengine - INFO - Current val iou of sem_cls is [ 0.889  0.061  0.472  0.025  0.083  0.085  0.327  0.247  0.060  0.128
> > 0.149  0.075]
> > 2025/06/27 16:35:24 - mmengine - INFO - Current val iou of sem is 0.15560498508398263
> > 2025/06/27 16:35:24 - mmengine - INFO - Current val iou of geo is 0.25096069999079246
> 
> We did not conduct experiments on NYUv2 since it only contains individual images and is not compatible with our embodied setting. For training local occupancy prediction on NYUv2, I suggest considering the following directions:
> 
> 1. Depth-aware branch: The depth prediction module we provide is tailored for the ScanNet dataset, so it may require adaptation for use with NYUv2.
> 2. Dataset-specific characteristics: Different datasets have different intrinsic properties, such as category distribution and so on. You can try adjusting the Gaussian parameters, the loss weightings, and other related settings to better suit this dataset.

Thanks for your valuable suggestions! I've tried to use the GT depth for NYUv2 to exam whether it is the problem with the depth prediction module. However, we only get 17. miou. Therefore, I guess it may not be the key reason. I will try to adjust other hyper-parameters.
